userInformation_program.c

```diff
@@ -0,0 +1,208 @@
+#include<stdio.h>
+#include<stdbool.h>
+#include<stdlib.h>
+#include<string.h>
+
+struct MyStructure
+{
+    int ID;
+    char name[100];
+    int age;
+};
+
+void create(FILE *file){
+    struct MyStructure s;
+    printf("\nEnter your ID:\n");
+    scanf("%d",&s.ID);
+    printf("\nEnter your name:\n");
+    scanf("%s",&s.name);
+    printf("\nEnter your age:\n");
+    scanf("%d",&s.age);
+
+    // setting file pointer to end
+    fseek(file,0,SEEK_END);
+    fwrite(&s,sizeof(s),1,file);
+
+    printf("\n user created. \n");
+
+}
+
+void read(FILE *file){
+    struct MyStructure s;
+
+    // setting file pointer to start
+    fseek(file, 0, SEEK_SET);
+    
+
+    int count=1;
+    while (fread(&s,sizeof(s),1,file))
+    {   
+        printf("\t\tUser-%d\n",count++);
+        printf("ID=%d\nName=%s\nAge=%d\n\n",s.ID,s.name,s.age);
+    }
+
+}
+
+void deleteUser(FILE *file){
+    int idToDelete;
+    printf("Enter user's ID you want to delete: ");
+    scanf("%d", &idToDelete);
+
+    FILE *temp = fopen("temp.txt", "w");
+    struct MyStructure s;
+
+    // setting file pointer to start
+    fseek(file, 0, SEEK_SET);
+    
+
+    bool flag=false;
+
+    while (fread(&s, sizeof(s), 1, file)) {
+        if (s.ID == idToDelete) {
+            flag = true;
+            //don't add this  
+        } else {
+            fwrite(&s, sizeof(s), 1, temp);
+        }
+    }
+
+    fclose(temp);
+    fclose(file);
+
+    if (flag==true) {
+        remove("users.txt"); 
```

Comment:
Too much use of inbuilt functions. 
think How you can make your own logic and use those functions here.